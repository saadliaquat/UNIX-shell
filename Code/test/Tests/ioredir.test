ls > ls.log
rm -f ls.log
cat ../Dockerfile > Dockerfile.log
ls > ls.log
cat ls.log
rm -f ls.log
cat Dockerfile.log
rm -f Dockerfile.log
echo "hello world" > file.txt
cat file.txt
echo "hello world 2" >> file.txt
cat file.txt
rm -f file.txt