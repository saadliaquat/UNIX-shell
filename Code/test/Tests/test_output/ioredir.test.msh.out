Dockerfile.log
Tests
config.json
config.json.log
find.log
ls.log
names.txt
test.py
hello world
hello world
hello world 2
